src/planner/binder/statement/bind_create_table.cpp

```diff
@@ -168,11 +319,48 @@ void Binder::BindDefaultValues(vector<ColumnDefinition> &columns, vector<unique_
 	}
 }
 
+void ReplaceGeneratedColumnReferences(unique_ptr<ParsedExpression> &expr, const BoundCreateTableInfo &table) {
+	if (expr->type == ExpressionType::COLUMN_REF) {
```

Comment:
Do we need to manually do this unrolling, or can the `CheckBinder` do this for us as part of the binding process?